### ISSUE TITLE
Edit repository details

### ISSUE THREAD

**Issue by RobinRMC** [hxr404/Discord-Console-hacks#73]
1. Change the description to `A collection of scripts to manipulate features of Discord and enhance the user experience.`<br>
    - Reason: the old description describes what may have been the case, but this is not accurate anymore. Not everything was made by you and not every script's purpose is to enhance the user experience, so I think the description I suggest provides a more accurate description of what this repository is actually about.<br><br>
2. Untick `Releases`, `Packages` and `Environments`<br>
    - Reason: these options don't get used in this repository, and thus create unnecessary clutter.

**Comment by hxr404** [comment]
1. Yeah you're right, the current description is outdated.
Your new description is better, but it lacks the part where the "Inner Workings of Discord" section is mentioned. Even though said section is considerably smaller than the script section, I think it's equally important, so that should kinda be reflected in the description.

2. Oh lol, actually I thought I did that already, GitHub moment :D
Thanks for the heads-up!

**Comment by RobinRMC** [comment]
It's near the repository description. There should be a gear icon (-> ⚙️).

**Comment by RobinRMC** [comment]
I'll look into the repository description sometime when I'm not laying in my bed.

**Comment by hxr404** [comment]
> It's near the repository description. There should be a gear icon (-> ⚙️).

yep thanks found it, seems like that thing is only displayed if you enable desktop view (I'm on mobile rn)

_I love GitHub's mobile-first frontend design /s_
